packs/equipment/holy-prayer-beads-greater.json

```diff
@@ -9,7 +9,7 @@
         },
         "containerId": null,
         "description": {
-            "value": "<p>This strand of ordinary-looking prayer beads glows with a soft light and becomes warm to the touch the first time you cast a divine spell while holding it. When you do, the prayer beads become attuned to your deity, changing their form and iconography to prominently incorporate your deity's religious symbol and iconography. The beads don't transform or function for an evil spellcaster.</p>\n<p>Whenever you cast a divine spell from your own spell slots while holding the prayer beads, you recover [[/r 1d4[vitality,healing]]]{1d4 Hit Points}; this is a vitality healing effect. If the spell you cast was a healing spell, you can grant this additional healing to one of the spell's targets instead of yourself.</p>\n<hr />\n<p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Effect</strong> Cast 4th-rank versions of @UUID[Compendium.pf2e.spells-srd.Item.Bless], @UUID[Compendium.pf2e.spells-srd.Item.Divine Wrath], @UUID[Compendium.pf2e.spells-srd.Item.Heal], or @UUID[Compendium.pf2e.spells-srd.Item.Cleanse Affliction] each once per day.</p>\n<hr />\n<p><strong>Craft Requirements</strong> You have a spellcasting class feature with the divine tradition.</p>"
+            "value": "<p>This strand of ordinary-looking prayer beads glows with a soft light and becomes warm to the touch the first time you cast a divine spell while holding it. When you do, the prayer beads become attuned to your deity, changing their form and iconography to prominently incorporate your deity's religious symbol and iconography. The beads don't transform or function for an evil spellcaster.</p>\n<p>Whenever you cast a divine spell from your own spell slots while holding the prayer beads, you recover @Damage[1d4[vitality,healing]]{1d4 Hit Points}; this is a vitality healing effect. If the spell you cast was a healing spell, you can grant this additional healing to one of the spell's targets instead of yourself.</p><hr /><p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Effect</strong> Cast 4th-rank versions of @UUID[Compendium.pf2e.spells-srd.Item.Bless], @UUID[Compendium.pf2e.spells-srd.Item.Divine Wrath], @UUID[Compendium.pf2e.spells-srd.Item.Heal], or @UUID[Compendium.pf2e.spells-srd.Item.Cleanse Affliction] each once per day.</p><hr /><p><strong>Craft Requirements</strong> You have a spellcasting class feature with the divine tradition.</p>"
```

Comment:
```suggestion
            "value": "<p>This strand of ordinary-looking prayer beads glows with a soft light and becomes warm to the touch the first time you cast a divine spell while holding it. When you do, the prayer beads become attuned to your deity, changing their form and iconography to prominently incorporate your deity's religious symbol and iconography. The beads don't transform or function for an evil spellcaster.</p>\n<p>Whenever you cast a divine spell from your own spell slots while holding the prayer beads, you recover @Damage[1d4[vitality,healing]] Hit Points; this is a vitality healing effect. If the spell you cast was a healing spell, you can grant this additional healing to one of the spell's targets instead of yourself.</p><hr /><p><strong>Activate</strong> Cast a Spell</p>\n<p><strong>Effect</strong> Cast 4th-rank versions of @UUID[Compendium.pf2e.spells-srd.Item.Bless], @UUID[Compendium.pf2e.spells-srd.Item.Divine Wrath], @UUID[Compendium.pf2e.spells-srd.Item.Heal], or @UUID[Compendium.pf2e.spells-srd.Item.Cleanse Affliction] each once per day.</p><hr /><p><strong>Craft Requirements</strong> You have a spellcasting class feature with the divine tradition.</p>"
```